src/ui/camera.js

```diff
@@ -855,7 +855,17 @@ class Camera extends Evented {
      * @returns A Boolean indicating whether the camera is moving.
      */
     isMoving(): boolean {
-        return this.moving;
+        return !!this.moving;
```

Comment:
Since `moving` and `zooming` are both boolean members of the Camera class already, let's remove the double negation here and below and just return `this.moving` / `this.zooming`.

Reply:
yep, `this.zooming` wasn't initialized in the constructor, so it could be `undefined` but I've initialized it as false in a new commit and removed the `!!`